projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { DetailRecord } from '@rero/ng-core/lib/record/detail/view/detail-record';
+import { Observable } from 'rxjs';
+import { TranslateService } from '@ngx-translate/core';
+import { RecordService } from '@rero/ng-core';
+import { UserService } from '../../../service/user.service';
+
+@Component({
+  selector: 'admin-document-detail-view',
+  templateUrl: './document-detail-view.component.html',
+  styles: []
+})
+export class DocumentDetailViewComponent implements DetailRecord, OnInit {
+  /** Observable resolving record data */
+  record$: Observable<any>;
+
+  /** Resource type */
+  type: string;
+
+  /** Document record */
+  record: any;
+
+  /** Holdings document */
+  holdings: any;
+
+  constructor(
+    private translateService: TranslateService,
+    private recordService: RecordService,
+    private userService: UserService
+    ) { }
+
+  ngOnInit() {
+    this.record$.subscribe((record: any) => {
+      this.loadHoldings(record.metadata.pid);
+      if ('authors' in record.metadata) {
+        record.metadata.authors = this.processAuthors(
```

Comment:
instead of modifying metadata, I would do a get method for authors:

```
get authors() {...}
```
And the same for identifiedBy and provisionActivity

Reply:
done